src/components/Container.jsx

```diff
@@ -0,0 +1,17 @@
+import Header from "./Header";
+import Main from "./Main";
+import Footer from "./Footer";
+import Nav from "./Nav";
+
+const Container = () => {
+  return (
+    <>
+      <Header />
```

Comment:
This might cause you some problems when you come to adding routes, as you are rendering everything inside this container :)